pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/QueryRunner.java

```diff
@@ -87,7 +87,7 @@ public void init(PinotConfiguration config, InstanceDataManager instanceDataMana
       _serverExecutor = new ServerQueryExecutorV1Impl();
       _serverExecutor.init(config, instanceDataManager, serverMetrics);
       _workerExecutor = new WorkerQueryExecutor();
-      _workerExecutor.init(config, serverMetrics, _mailboxService, _hostname, _port);
+      _workerExecutor.init(_mailboxService, _hostname, _port);
```

Comment:
why reducing the init method signature?

Reply:
these weren't used - so just some cleanup. I can remove it if I'm missing something

Reply:
oops! sorry I thought I had done that but looks like I only fixed the other code path 😬 shame on me